Update plugin deprecation page (#7022)
jenkins-infra/jenkins.io: content/doc/developer/plugin-governance/deprecating-or-removing-plugin.adoc

  title: List of the deprecated Jenkins plugins
---

> Some plugins may be replaced by others or just become irrelevant (e.g. integration with a service which was shut down or a new plugin combining multiple plugins into one).

Reviewer: ```suggestion
Some plugins may be replaced by others or just become irrelevant.
For example, becoming irrelevant can include an integration with a service that was shut down or a new plugin combining multiple plugins into one.
```
Suggestion to remove the usage of `e.g.` based on the [Microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg).

Author: Learning something new every day, thanks!

Reviewer 2: ... So you replaced it with another term "for example" that Microsoft suggest to avoid cf your link? 😅

   Choose this approach if the GitHub repository contains multiple plugins and only some of the plugins in the repository are to be deprecated.
. Update the plugin's documentation to explain the reason of the deprecation.

== Archiving the repository

Optionally, you can archive the plugin's GitHub repository, if the plugin is end of life or superseded by another plugin.

> . Leave a note in the repository's README file explaining the reason for archiving and pointing to the replacement plugin if available.

Reviewer 3: ```suggestion
. Leave a note in the repository's README file explaining the reason for archiving and providing a link to the replacement plugin if available.
```
